packages/core/src/internal/utils/events.ts

```diff
@@ -1,9 +1,11 @@
 /*
- * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
+import { isJestTest } from './environment';
```

Comment:
I think this import needs to come from `./environment.js`...

Reply:
vscode auto import, oops. Fixed